.github/workflows/release.yaml

```diff
@@ -0,0 +1,51 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+env:
+  go-version: 1.16
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Setup golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+
+      - name: Build binaries
+        run: make install GOBIN=`pwd`/docker
+
+      - name: Build setup-hw image
+        run: |
+          docker build -t quay.io/cybozu/setup-hw:latest --target stage1 docker
+          docker build -t quay.io/cybozu/setup-hw-secret:latest docker
+
+      - name: Push images
+        run: |
+          echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USER }} --password-stdin quay.io
+          TAG=${GITHUB_REF#refs/tags/v}
+          for name in setup-hw setup-hw-secret; do
+              echo "pushing ${name}:v${TAG} ..."
+              docker tag quay.io/cybozu/${name}:latest quay.io/cybozu/${name}:$TAG
+              docker push quay.io/cybozu/${name}:latest
+              docker push quay.io/cybozu/${name}:$TAG
+          done
+
+      - name: Install ghr
+        run: go get -u github.com/tcnksm/ghr
+
+      - name: Create release
+        run: |
+          OWNER=$(echo ${{ github.repository }} | cut -d '/' -f 1)
+          REPOSITORY=$(echo ${{ github.repository }} | cut -d '/' -f 2)
+          if ${{ env.prerelease }}; then
+            PRERELEASE="-prerelease"
+          fi
+          TAG=${GITHUB_REF#refs/tags/}
+          ghr -t ${{ secrets.GITHUB_TOKEN }} -u ${OWNER} -r ${REPOSITORY} -n "Release v${TAG}"" ${PRERELEASE} -b "See [CHANGELOG.md](./CHANGELOG.md) for details." ${TAG}
```

Comment:
Please use [`gh`](https://github.com/cli/cli).
It is pre-installed on GitHub actions and it is an official tool.

Example: https://github.com/cybozu-go/moco/blob/main/.github/workflows/release.yaml#L35